packages/console/src/log/LogView.tsx

```diff
@@ -96,15 +103,29 @@ class LogView extends PureComponent {
     }
   }
 
-  componentWillUnmount() {
+  componentWillUnmount(): void {
     this.stopFlushTimer();
     this.stopListening();
     this.destroyMonaco();
 
     window.removeEventListener('resize', this.handleResize);
   }
 
-  getMenuActions(shownLogLevels) {
+  cancelListener?: () => void | null;
+
+  editor?: monaco.editor.IStandaloneCodeEditor;
+
+  editorContainer: HTMLDivElement | null;
+
+  logLevelMenuItems: Record<string, LogLevelMenuItem | null>;
```

Comment:
I don't think need to track `null` here. Just check if `element != null` before setting.
```suggestion
  logLevelMenuItems: Record<string, LogLevelMenuItem>;
```